app/src/components/integrations/elements/integrationSettings/integrationSettings.jsx

```diff
@@ -62,6 +63,18 @@ export const IntegrationSettings = (props) => {
   const availableProjectIntegrations = projectIntegrations[query.subPage] || [];
   const groupedIntegrations = [...availableGlobalIntegrations, ...availableProjectIntegrations];
 
+  const {
+    data,
+    onUpdate,
+    formFieldsComponent,
+    editAuthConfig,
+    isEmptyConfiguration,
+    formKey,
+    isGlobal,
+  } = props;
+
+  const pluginName = props.data.integrationType?.name;
```

Comment:
Looks like it is better to extract plugin name from data using destructuring.
 const {
    data,
    onUpdate,
    formFieldsComponent,
    editAuthConfig,
    isEmptyConfiguration,
    formKey,
    isGlobal,
    data:{integrationType:{name:pliginName}}={}
  } = props;

Reply:
Reverted all changes